### DISCUSSION TITLE
My copilot account under waitlist for the past three weeks when can it be get approved?

### DISCUSSION THREAD

**Discussion by Yerramilli0429** [community/community#9166] [Copilot Conversations] [answered]
Till my Copilot account under wait list for the past three weeks when can it be approved?

**Comment by yamin8000** [comment]
> Till my Copilot account under wait list for the past three weeks when can it be approved?

for me, it took a couple of months!

**Comment by Maiveasna** [comment]
Copilot account under wait list for the past four weeks. when can it be approved?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
